lib/prehog/proto/prehog/v1alpha/teleport.proto

```diff
@@ -103,17 +94,24 @@ message SubmitEventRequest {
     SessionStartEvent session_start = 6;
 
     UIBannerClickEvent ui_banner_click = 7;
-    UIOnboardGetStartedClickEvent ui_onboard_get_started_click = 8;
     UIOnboardCompleteGoToDashboardClickEvent ui_onboard_complete_go_to_dashboard_click = 9;
     UIOnboardAddFirstResourceClickEvent ui_onboard_add_first_resource_click = 10;
     UIOnboardAddFirstResourceLaterClickEvent ui_onboard_add_first_resource_later_click = 11;
     UIOnboardSetCredentialSubmitEvent ui_onboard_set_credential_submit = 12;
     UIOnboardRegisterChallengeSubmitEvent ui_onboard_register_challenge_submit = 13;
     UIRecoveryCodesContinueClickEvent ui_recovery_codes_continue_click = 14;
+    UIRecoveryCodesCopyClickEvent ui_recovery_codes_copy_click = 15;
+    UIRecoveryCodesPrintClickEvent ui_recovery_codes_print_click = 16;
   }
+  reserved 8; // UIOnboardGetStartedClickEvent
+  reserved "ui_onboard_get_started_click";
 }
 message SubmitEventResponse {}
 
+message HelloTeleportRequest {}
+message HelloTeleportResponse {}
+
 service TeleportReportingService {
   rpc SubmitEvent(SubmitEventRequest) returns (SubmitEventResponse) {}
+  rpc HelloTeleport(HelloTeleportRequest) returns (HelloTeleportResponse) {}
```

Comment:
Not seeing any of `HelloTeleport`, `HelloTeleportRequest`, or `HelloTeleportResponse` being used. Are they necessary for a future PR?

Reply:
We may; Edoardo added these RPCs to [prehog in November](https://github.com/gravitational/prehog/pull/39). While we leveraged the check in the [Cloud repo](https://github.com/gravitational/cloud/blob/b76786f7d131ed26ecf666f2b3e83666745d6aa4/tools/salescenter/process/process.go#L271), we haven't yet in Teleport. @timothyb89, what are your thoughts on adding this check? 

We're seeing this update here today because I'm copying over the proto for the first time in a minute, and it's brining both my changes and any other changes that have occurred since the last time we manually updated the contract.

Reply:
I think my preference would be to keep the RPC here just so we don't have to manually edit the upstream proto from prehog.